src/System.Windows.Forms/tests/IntegrationTests/WinformsControlsTest/TrackBars.cs

```diff
@@ -29,7 +29,17 @@ private void chbRightToLeft_CheckedChanged(object sender, EventArgs e)
             trackBar1.RightToLeft = chbRightToLeft.Checked ? RightToLeft.Yes : RightToLeft.No;
         }
 
-        private void numericMinimum_ValueChanged(object sender, EventArgs e)
+        private void tickstyleNone_CheckedChanged(object sender, EventArgs e)
+        {
+            if (trackBar1.TickStyle.Equals(System.Windows.Forms.TickStyle.BottomRight))
```

Comment:
A more C#-like would be this:
```suggestion
            if (trackBar1.TickStyle == TickStyle.BottomRight)
```

...and use `using System.Windows.Forms;` added at the top of the document.